### PR TITLE
Add cluster size constraint to template UI

### PR DESCRIPTION
**Pull request by gaarf**
Fixes #425

### CODE REVIEW

**Comment by wolf31o2** [comment]
This LGTM...

File: ui/client/templates/user/clusters/createcluster.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
             <div class="form-group">
               <label for="inputNumMachines" class="col-sm-2 control-label">Number of machines</label>
               <div class="col-sm-8">
-                <input type="text" class="form-control" id="inputNumMachines" name="inputNumMachines" placeholder="Machines" ng-model="clusterNumMachines">
+                <input type="number" class="form-control" id="inputNumMachines" name="inputNumMachines" placeholder="Machines" ng-model="clusterNumMachines" min="[[template.constraints.size.min]]" max="[[template.constraints.size.max]]">
```
Comment (wolf31o2) [inline ui/client/templates/user/clusters/createcluster.html:49]: Can this default to `template.constraints.size.min` so we end up with one less click/user input?

Comment (gaarf) [inline ui/client/templates/user/clusters/createcluster.html:49]: sure

**Comment by wolf31o2** [comment]
:+1:

**Pull request merged**